### DISCUSSION TITLE
Cara Penggunaan Library pada Controller

### DISCUSSION THREAD

**Discussion by nor-arifin** [ivanwilliammd/satusehat-integration#121] [Q&A (Tanya Jawab)] [answered]
Saya membuat PatientController

public function create(){

        $client = new OAuth2Client;
        $token = $client->token(); // OAuth2Token anda akan muncul

        return view('pages.patients.create', compact('token'));
    }
saya bermaksud ingin echo $token pada view,
muncul error berikut :
Typed property Satusehat\Integration\OAuth2Client::$organization_id must not be accessed before initialization

Apakah ada kesalahan penggunaan di saya ?
![Screenshot 2024-03-22 234957](https://github.com/ivanwilliammd/satusehat-integration/assets/155248517/aac3b3f8-8e9c-4467-bdcf-0d79b193f87a)

**Comment by ivanwilliammd** [comment]
Untuk organization_id seharusnya diinisialisasi pada .env

**Comment by ivanwilliammd** [comment, marked as answer]
Note : cara penggunaan bisa merujuk pada link ini https://github.com/ivanwilliammd/satusehat-laravel-example
Setelah mengisi data pada .env coba akses http://127.0.0.1:8000/token
